tests/test_bgp_af.rb

```diff
@@ -269,22 +330,17 @@ def test_additional_paths
     # Do a 'additional-paths send, receive, install'
     bgp_af.additional_paths_send = true
     bgp_af.additional_paths_receive = true
-    bgp_af.additional_paths_install = true
-
-    af_string = get_bgp_af_cfg(asn, vrf, af)
-
-    [pattern_send, pattern_receive, pattern_install].each do |pat|
-      assert_match(pat, af_string,
-                   "Error: '#{pat}' is not configured and should be")
-    end
+    bgp_af.additional_paths_install = true if platform == :nexus
```

Comment:
Should we set these for both platforms and then catch the error? Ditto to similar.